### ISSUE TITLE
Mysterious Move Database Self Destruct Mode

### ISSUE THREAD

**Issue by jofomah** [eHealthAfrica/move#159]
@channel: Apparently, our Move app is in self destruct mode, from missing user accounts to, Facility document email being changed to wrong email address(which AFAIK is not being updated by any part of the app, should be read only, the document revisions showed this was updated several times after it was created), prompting some users for password even after they have been configured, recently we noticed that 20+ facilities were not configured during the training that is our assumption though these facilities have user accounts but they don't have app configs

Yesterday, a facility app config product profile was set to empty array, hence the facility couldn't send in stock count, when i checked the app config, it showed the facility was activated on 13th July, a week after the training and these facilities were all setup setup before 13th July, my guess, the original app config was deleted, and the app config was reconfigured on 13th July. but maybe that explains the missing product profiles, which is preset by default and can't be empty unless deselected and the users have to tap five times on the app config to enable deselection etc

long story short, Norbert please could you help us pull the database server logs from 6th July, 2015

maybe we can find a pattern, or understand what the root cause of these could be, like was the 26 facilities configured and then accidentally deleted? was the facility app config updated or recreated on 13th July? maybe the IP address of the request etc (edited)

anything that can help on identifying the possible bug or whatever it is that messes with the user access, user accounts, deletion and making unwanted updates to the database. (edited)

also, Justin maybe we need to deploy Couch Changes Saver for LoMIS database

**Comment by tlvince** [comment]
@jofomah was this resolved?

**Comment by jofomah** [comment]
i will say Yes, it wasn't due to the mobile app, more of a process thing.
